exercises/jigsaw-puzzle/introduction.md

```diff
@@ -0,0 +1,4 @@
+# Introduction
+
+Your best friend has started collecting jigsaw puzzles and wants to build a detailed catalog of their collection — recording information such as the total number of pieces, the number of rows and columns, the number of border and inside pieces, aspect ratio, and puzzle format.
+After solving one of the puzzles to figure out these details, you're now helping fill in the gaps for others in the collection, even when some data is missing.
```

Comment:
I'm not great at this, but maybe this could be "spiced up" a little? To make it feel a little more whimsical.

Reply:
I tried my best.